### PR TITLE
more ruff auto-fixes

### PR DESCRIPTION
**Pull request by fchapoton**
A few more fixes, done using

`ruff check --fix --select=UP008,UP034,UP031 flatsurf`

then `black`

Checklist
* [ ] Added an entry in `doc/news/`. 
* [ ] Added a test for this change.
* [ ] Added new .py files to the documentation in `doc/geometry` or `doc/graphical`.

### CODE REVIEW

**Comment by fchapoton** [comment]
another one ready to go

**Comment by saraedum** [comment]
Can you add these to https://github.com/flatsurf/sage-flatsurf/blob/master/.github/workflows/lint.yml#L75?

**Comment by fchapoton** [comment]
Voilà. I also remove the `--fix` in the call to `ruff`, which is not useful when checking.

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ jobs:
       - name: Run ruff
         shell: bash -l {0}
         run: |
-          ruff check --fix --select C4 .
+          ruff check --select C4,UP008,UP031,UP034 .
       - name: Detect changes
         shell: bash -l {0}
         run: git diff --exit-code
```
Comment (saraedum) [inline .github/workflows/lint.yml:78]:
```suggestion
```

**Pull request merged**